src/Algolia.Search/Models/Search/PartialUpdateOperation.cs

```diff
@@ -57,6 +57,34 @@ public static PartialUpdateOperation<int> Increment(int value)
             };
         }
 
+        /// <summary>
+        /// Increment a numeric integer attribute only if the provided value matches the current value, and otherwise ignores the whole object update
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
```

Comment:
nit: 

- `<returns>` can be removed in unused
- same goes for `param name="value"></param>` if the parameter `values` isn't described in the tag.